src/components/Card.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Card from './Card';
+import { shallow } from 'enzyme';
+
+
+describe('Card', () => {
+  test('matches a snapshot', () => {
+    const wrapper = shallow(<Card id={2} deleteCardCallback={() => {}} />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
This is good, however we could probably include more snapshot tests.

Specifically, we frequently want to have snapshot tests for each of the props that a component receives which have some impact on the visual result. In this case that would be the `text` and `emoji` props.

Having at least one test for each of those props (and maybe one for when both are set), will help us to cover more possible outcomes from rendering a component. This will in turn help ensure that we don't accidentally break the rendering for some part of the component.

As an example, if we accidentally removed line 25 from `Card.js` (`<p className="card_content-text">{this.props.text}</p>`) then our card would not render correctly, but this test might continue to pass.